manifests/script-lab-edge.xml

```diff
@@ -17,7 +17,11 @@
     <AppDomain>https://github.com</AppDomain>
     <AppDomain>https://stackoverflow.com</AppDomain>
     <AppDomain>https://dev.office.com</AppDomain>
-    <AppDomain>https://bornholm-runner-edge.azurewebsites.net</AppDomain>  
+    <AppDomain>https://script-lab.azureedge.net</AppDomain>
+    <AppDomain>https://script-lab-runner.azureedge.net</AppDomain>
+    <AppDomain>https://bornholm-runner-edge.azurewebsites.net</AppDomain>
```

Comment:
Added the same AppDomains to every manifest, with the exception of `localhost` and its runner domain. Do we want to have `localhost` be an option for the user to switch back to or no?